extensions/control-plane/api/management-api/contract-negotiation-api/src/main/java/org/eclipse/edc/connector/api/management/contractnegotiation/ContractNegotiationApi.java

```diff
@@ -132,8 +132,6 @@ record ContractRequestSchema(
             String connectorAddress,
             @Schema(requiredMode = REQUIRED)
             String counterPartyAddress,
-            @Schema(requiredMode = REQUIRED)
-            String providerId,
```

Comment:
do not delete but mark it as deprecated with a short description

Reply:
reverted.